src/App.js

```diff
@@ -1,14 +1,11 @@
-import Container from 'react-bootstrap/Container';
 import './App.css';
 import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <Container>
-      <div className="App">
-        <Calculator />
-      </div>
-    </Container>
+    <div className="App">
+      <Calculator />
+    </div>
```

Comment:
- [ ] Lastly, I noticed that after I clicked on a number and I wanted to click on another number, they all didn't work except `7`. But after I use one of the symbols that is when I will be able to click on another number and even after clicking on the number, I can't click on any other number except `7`. But as you know, any calculator should be able to process a variety of computations with various numbers so kindly fix this as well. I have attached a video reference below so you have a look :+1: 
- [ ] You can also see from the video below that your calculator doesn't have the number `2` so kindly this as  well :+1: 

- Video Reference
[Screencast from 03-29-2023 05:19:58 PM.webm](https://user-images.githubusercontent.com/102699958/228618242-6a516494-5bd6-40c7-9d29-21f2d53d14b4.webm)